### ISSUE TITLE
Online high scores

### ISSUE THREAD

**Issue by cxong** [cxong/FallingTime#4]
Possibly using [Compo4All](http://boards.dingoonity.org/gcw-development/sparrow-compo4all-manager/)

**Comment by akien-mga** [comment]
Fixed by 362325e?

**Comment by cxong** [comment]
no, that only covers local high scores. I was hoping to get online high scores too.
